source/lib.civet

```diff
@@ -130,21 +130,23 @@ function updateParentPointers(node: ASTNode, parent?: ASTNodeBase, depth = 1): v
   }
 }
 
-function addPostfixStatement(statement: StatementTuple, ws: ASTNode, post) {
+function addPostfixStatement(statement: StatementTuple, ws: ASTNode, post)
   let children, expressions
-  if (post.blockPrefix?.length) {
-    let indent = post.blockPrefix[0][0]
-    expressions = [...post.blockPrefix, [indent, statement]]
+
+  prefix := post.blockPrefix or post.condition?.expression.blockPrefix
```

Comment:
What if they both exist?  This might trigger it:
```js
(if x := f()
  y
else
  z
) if w := g()
```

Reply:
IfExpressions don't currently handler declaration condition at all, we need to statementize them first.